oioswift/common/middleware/container_hierarchy.py

```diff
@@ -440,7 +440,7 @@ def _remove_placeholder(self, req, account, container, mode, path):
                             None,
                             limit=1,
                             force_master=True,
-                            versions=self.support_listing_versioning))
+                            versions=True))
```

Comment:
We need to implement an `is_empty` call in oio-proxy, which would be lighter than listing objects (meta2 already supports this call, but it is not exposed).

Reply:
:+1: